packages/dds/merge-tree/src/test/client.localReference.spec.ts

```diff
@@ -197,6 +199,80 @@ describe("MergeTree.Client", () => {
 		);
 	});
 
+	describe("Slides on first ack of remove/obliterate when both are present", () => {
+		// This suite demonstrates that even when we decide we don't need to emit events for segments going away (since
+		// they have already been removed locally), we still need to slide local references that are on those segments.
+		// Failure to do so can break eventual consistency in features relying on SlideOnRemove references (such as
+		// IntervalCollection).
+		it("Remove then move", () => {
+			const helper = new ReconnectTestHelper({
+				mergeTreeEnableSidedObliterate: true,
```

Comment:
Does this not fail if sided obliterate is enabled without the regular `mergeTreeEnableObliterate`?

Reply:
yeah the specific flags we pass here don't really end up mattering as long as we have *some* obliterate that we can use.